src/genie/libs/parser/iosxe/show_vrf.py

```diff
@@ -62,7 +62,7 @@ def cli(self, vrf='', output=None):
         # vpn4                             100:2                 ipv4,ipv6
         # rb-bcn-lab                       10.116.83.34:1        ipv4,ipv6   Lo9
         # test                             10.116.83.34:100      ipv4,ipv6   Lo100
-        p1 = re.compile(r'^(?P<vrf>[\w\d\-\.]+)\s+(?P<rd>\<not +set\>|[\.\d\:]+)\s+'
+        p1 = re.compile(r'^(?P<vrf>[\w\d\-\.\:]+)\s+(?P<rd>\<not +set\>|[\.\d\:]+)\s+'
```

Comment:
can I ask you to add testing for the parser?

reference:
https://pubhub.devnetcloud.com/media/pyats-development-guide/docs/writeparser/writeparser.html#testing-your-parser

Reply:
I did. Hopefully it will parse :-) it did on real devices.